Source/ERF.cpp

```diff
@@ -84,6 +85,15 @@ Vector<std::string> BCNames = {"xlo", "ylo", "zlo", "xhi", "yhi", "zhi"};
 //             - initializes BCRec boundary condition object
 ERF::ERF ()
 {
+    int fix_random_seed = 0;
+    ParmParse pp("erf"); pp.query("fix_random_seed", fix_random_seed);
+    // Note that the value of 1024UL is not significant -- the point here is just to set the
+    // same seed for all MPI processes for the purpose of regression testing
+    if (fix_random_seed) {
+        Print() << "Fixing the random seed" << std::endl;
+        InitRandom(1024UL);
+    }
+
```

Comment:
@asalmgren Is this okay? I moved this piece of code from `Initialization/ERF_InitCustom.cpp` to the beginning of the ERF constructor.

Reply:
Yes -- happy to have it there.    I'm pushing through some PRs today to do live regression so I'll merge this and some others very soon.